boards/riscv/mpfs_icicle/mpfs_icicle_defconfig

```diff
@@ -15,8 +15,10 @@ CONFIG_RISCV_SOC_INTERRUPT_INIT=y
 CONFIG_RISCV_HAS_PLIC=y
 CONFIG_PLIC=y
 CONFIG_RISCV_MACHINE_TIMER=y
-CONFIG_GPIO=n
 CONFIG_XIP=n
 CONFIG_INIT_STACKS=y
 CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
 CONFIG_FPU=n
+# GPIO driver options
+CONFIG_GPIO=y
+CONFIG_GPIO_MCHP_MSS=y
```

Comment:
remove `CONFIG_GPIO_MCHP_MSS=y` should get handled automatically by Kconfig

Reply:
Hi @galak,
Thanks for reviewing. Have finished testing requested changes locally and will push now.